OneSignalSDK/unittest/src/test/java/com/test/onesignal/RemoteParamsTests.java

```diff
@@ -150,7 +150,9 @@ public void testUserPrivacyConsentRequired_UserConfigurationOverrideByRemotePara
 
     @Test
     public void testUserPrivacyConsentNotRequired_UserConfigurationOverrideByRemoteParams() throws Exception {
-        ShadowOneSignalRestClient.setRemoteParamsGetHtmlResponse();
+        JSONObject remoteParams = new JSONObject();
+        remoteParams.put("requires_user_privacy_consent", false);
```

Comment:
there is a method for doing this under ShadowOneSignalRestClient

`setRemoteParamsRequirePrivacyConsent`

Reply:
Updated